homework1/1.2/1.2/1.2.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+
+namespace BurrowsWheelerTransform
+{
+    class Program
+    {
+        private static string BurrowsWheelerTransform(string originalString, ref int index)
+        {
+            string[] tableOfStrings = new string[originalString.Length];
```

Comment:
```suggestion
            var tableOfStrings = new string[originalString.Length];
```
Два раза тип на строке писать не надо